limitador-server/src/main.rs

```diff
@@ -290,12 +295,37 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .max_level_hint()
                 .unwrap_or(LevelFilter::ERROR)
         });
-        let builder = if level >= LevelFilter::DEBUG {
-            tracing_subscriber::fmt().with_span_events(FmtSpan::CLOSE)
+        let fmt_layer = if level >= LevelFilter::DEBUG {
+            tracing_subscriber::fmt::layer().with_span_events(FmtSpan::CLOSE)
         } else {
-            tracing_subscriber::fmt()
+            tracing_subscriber::fmt::layer()
+        };
+
+        if !config.tracing_host.is_empty() {
+            let tracer = opentelemetry_otlp::new_pipeline()
+                .tracing()
+                .with_exporter(
+                    opentelemetry_otlp::new_exporter()
+                        .tonic()
+                        .with_endpoint(format!("rpc://{}", config.tracing_address()))
+                        .with_protocol(Protocol::Grpc),
+                )
+                .with_trace_config(trace::config().with_resource(Resource::new(vec![
+                    KeyValue::new("service.name", "limitador-server"),
+                ])))
+                .install_batch(opentelemetry_sdk::runtime::Tokio)?;
+            let telemetry_layer = tracing_opentelemetry::layer().with_tracer(tracer);
+            tracing_subscriber::registry()
+                .with(level)
+                .with(fmt_layer)
+                .with(telemetry_layer)
+                .init();
+        } else {
+            tracing_subscriber::registry()
```

Comment:
what is this doing? (when exporter is not being defined)

Reply:
The tracing package is being used for the standard logging since #250 - if `tracing_host` is unset we ensure that the `fmt_layer` is still applied so we get log output on stdout